### ISSUE TITLE
不能使用Juggernaut-XL-v8和Juggernaut-X-v10？我用这两个模型出来的结果完全不对，而其他模型都是对的。

### ISSUE THREAD

**Issue by Amo5** [HVision-NKU/StoryDiffusion#136]
这是我的结果：
![image](https://github.com/HVision-NKU/StoryDiffusion/assets/41940277/0b689a0a-7f30-42b7-b5f6-f240dbf6c665)

红色框里面 是对应的prompt。感觉毫不相关。。我用Juggernaut-X-v10，打了个简单的T2I gradio，是完全没问题的：
说明我的权重是对的。下面是我的gradio代码。（然而，在StoryDiffusion 链路中，这两个模型生成图像都不对呢？）
`from diffusers import StableDiffusionXLPipeline
import torch
from gradio import Interface, Image, Dropdown, Slider
import gradio as gr
import spaces

model_id = "pretrained_models/RunDiffusion/Juggernaut-X-v10"
pipe = StableDiffusionXLPipeline.from_pretrained(model_id, torch_dtype=torch.float16)
pipe = pipe.to("cuda")

@spaces.GPU()
def text_to_image(prompt, negative_prompt, steps, guidance_scale, progress=gr.Progress(track_tqdm=True)):
    image = pipe(prompt, negative_prompt=negative_prompt, num_inference_steps=steps, guidance_scale=guidance_scale).images[0]
    return image
#duplicate_button = gr.DuplicateButton(value="Duplicate Space for private use", elem_id="duplicate-button")

gradio_interface = Interface(
    fn=text_to_image,
    allow_duplication=False,
    inputs=[
        gr.Textbox(label="Prompt", lines=2, placeholder="Enter your prompt here..."),
        gr.Textbox(label="Negative Prompt", lines=2, placeholder="What to exclude from the image..."),
        gr.Slider(minimum=1, maximum=65, value=50, label="Steps", step=1),
        gr.Slider(minimum=1, maximum=20, value=7.5, label="Guidance Scale", step=0.1)
    ],
    outputs=Image(type="pil", show_download_button=True),
    examples=[
        ["1 woman, 28 yo , full body pose, smiling , super realistic , happy girl, perfect skin, blonde hair, perfect face , perfect body , wearing a small white bikini panties, cameltoe, real life, skinny but fit , 4k, high quality, (masterpiece)"],
    ],
    cache_examples=False,
    theme=gr.themes.Soft()
)
gradio_interface.launch(share=False, server_name="127.0.0.1", server_port=7861,root_path='/proxy/7861/', enable_queue=True)`

**Comment by milely** [comment]
我用一些粘土风格的xl-based的模型出来也是效果不对，有一些是正确的，可能对某些模型不适配。
